elastic/apm-server
# [fleet integration] Support all apm-server config options and adapt defaults.

## Motivation/summary
* Add missing `apm-server.*` settings that should be supported to the APM Server Integration.
* Adapt default values to be aligned with cloud values where possible.




## Checklist




## How to test these changes
- Spin up a local package registry and copy over the `apmpackage` information to a new version `0.3.0`. 
- Spin up an elastic-agent 
- Add an apm-integration and ensure the new config values are there



## Related issues

implements #4528



## Preview
<img width="328" alt="Screenshot 2021-06-14 at 09 59 39" src="https://user-images.githubusercontent.com/5555349/121858832-4c0f6d80-ccf7-11eb-9e2d-ff8bf688b7fa.png">
<img width="294" alt="Screenshot 2021-06-14 at 09 59 46" src="https://user-images.githubusercontent.com/5555349/121858840-4f0a5e00-ccf7-11eb-817c-bd42fb49b779.png">
